gns3server/config.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, files=None, profile=None):
         self._files = files
         self._profile = profile
         if files and len(files):
+            directory_name = os.path.dirname(files[0])
+            if not directory_name or directory_name == "":
+                files[0] = './' + files[0]
```

Comment:
I would use files[0] = os.path.abspath(files[0]). This will make sure it works on Windows as well.

Reply:
I may have misunderstood what files[0] is supposed to represent in this case. 

Since we're assigning  `self._main_config_file = files[0]`, `files[0]` (and by extension `self._main_config_file`) is supposed to point to the actual configuration file and not a directory right? Or is the behaviour different on Windows?

Reply:
Correct files[0] points to the actual configuration file, not a directory. Actually, I think both solution would work. I am just a bit concerned that the './' would not work on Windows.